### PR TITLE
[MDCT-422] Section C: Program Integrity

### PR DESCRIPTION
**Pull request by juliuscaesar**
## Description
Adds Section C: Program Integrity. Closes [MDCT-422](https://qmacbis.atlassian.net/browse/MDCT-422)

### How to test
* Checkout the branch
* Run `./dev local`
* Navigate to http://localhost:3000/mcpar/program-level-indicators/program-integrity

### Changed Dependencies
N/A

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

**Review by karla-vm — APPROVED** [review]
Looks good! ✨

**Review by braxex — APPROVED** [review]
This is perfect. Just needs the changes from https://github.com/CMSgov/mdct-mcr/pull/3460 to work. Wait until that's merged, then merge in main, then you're good to merge this one. 👍

**Comment by braxex** [comment]
@juliuscaesar All updated with main and ready to merge at your convenience. 👍

**Pull request merged**
